plugins/action_join_user_org/src/joinUserOrg.ts

```diff
@@ -1,4 +1,8 @@
-const joinUserOrg = async function ({ applicationData, ...userOrg }: any, DBConnect: any) {
+import { ActionPluginInput } from '../../types'
+
+// const joinUserOrg = async function ({ applicationData, ...userOrg }: any, DBConnect: any) {
+const joinUserOrg = async function ({ parameters: userOrg, DBConnect }: ActionPluginInput) {
+  // const userOrg = parameters
```

Comment:
Also this line here

Reply:
It's just renaming the whole parameters object -- because the userOrg object gets sent to the database query as a whole object, I didn't think we needed to destructure it to individual values. But renaming it just makes it easier than seeing `parameters` throughout.